diff --git a/assets/i18n/zh-CN.json b/assets/i18n/zh-CN.json
index 0d8f584fc..e36db72ad 100644
--- a/assets/i18n/zh-CN.json
+++ b/assets/i18n/zh-CN.json
@@ -155,5 +155,6 @@
   "enable_orochi_ten_once": "攻击一次御魂十层",
   "pets_feast": "快速喂养",
   "costume_main_15": "茨心乐园",
-  "get_flower": "启用收取等级奖励"
+  "get_flower": "启用收取等级奖励",
+  "costume_main_16": "琦梦茨庭"
 }
\ No newline at end of file
diff --git a/tasks/Component/Costume/assets.py b/tasks/Component/Costume/assets.py
index d7d95fe16..49f693905 100644
--- a/tasks/Component/Costume/assets.py
+++ b/tasks/Component/Costume/assets.py
@@ -101,18 +101,31 @@ class CostumeAssets:
 
 
 	# Image Rule Assets
-	# 茨心乐园庭院判定
+	# 茨心乐园庭院判定 
 	I_CHECK_MAIN_15 = RuleImage(roi_front=(1017,365,71,48), roi_back=(900,270,300,220), threshold=0.75, method="Template matching", file="./tasks/Component/Costume/main15/main15_check_main_15.png")
-	# 探索
+	# 探索 
 	I_MAIN_GOTO_EXPLORATION_15 = RuleImage(roi_front=(590,190,30,50), roi_back=(520,130,190,180), threshold=0.55, method="Template matching", file="./tasks/Component/Costume/main15/main15_main_goto_exploration_15.png")
-	# 召唤屋
+	# 召唤屋 
 	I_MAIN_GOTO_SUMMON_15 = RuleImage(roi_front=(960,204,28,45), roi_back=(890,130,200,180), threshold=0.55, method="Template matching", file="./tasks/Component/Costume/main15/main15_main_goto_summon_15.png")
-	# 町中
+	# 町中 
 	I_MAIN_GOTO_TOWN_15 = RuleImage(roi_front=(838,324,24,49), roi_back=(760,250,200,190), threshold=0.8, method="Template matching", file="./tasks/Component/Costume/main15/main15_main_goto_town_15.png")
-	# 宠物屋
+	# 宠物屋 
 	I_PET_HOUSE_15 = RuleImage(roi_front=(951,383,31,50), roi_back=(900,330,150,150), threshold=0.8, method="Template matching", file="./tasks/Component/Costume/main15/main15_pet_house_15.png")
 
 
+	# Image Rule Assets
+	# 庭院 
+	I_CHECK_MAIN_16 = RuleImage(roi_front=(1017,365,71,48), roi_back=(900,270,300,220), threshold=0.75, method="Template matching", file="./tasks/Component/Costume/main16/main16_check_main_16.png")
+	# 探索 
+	I_MAIN_GOTO_EXPLORATION_16 = RuleImage(roi_front=(591,183,28,46), roi_back=(520,130,190,180), threshold=0.55, method="Template matching", file="./tasks/Component/Costume/main16/main16_main_goto_exploration_16.png")
+	# 召唤屋 
+	I_MAIN_GOTO_SUMMON_16 = RuleImage(roi_front=(966,199,23,40), roi_back=(890,130,200,180), threshold=0.55, method="Template matching", file="./tasks/Component/Costume/main16/main16_main_goto_summon_16.png")
+	# 町中 
+	I_MAIN_GOTO_TOWN_16 = RuleImage(roi_front=(839,330,23,43), roi_back=(760,250,200,190), threshold=0.8, method="Template matching", file="./tasks/Component/Costume/main16/main16_main_goto_town_16.png")
+	# 宠物屋 
+	I_PET_HOUSE_16 = RuleImage(roi_front=(954,371,28,39), roi_back=(900,330,150,150), threshold=0.8, method="Template matching", file="./tasks/Component/Costume/main16/main16_pet_house_16.png")
+
+
 	# Image Rule Assets
 	# description 
 	I_CHECK_MAIN_2 = RuleImage(roi_front=(1090,146,48,55), roi_back=(880,108,271,100), threshold=0.8, method="Template matching", file="./tasks/Component/Costume/main2/main2_check_main_2.png")
@@ -216,3 +229,4 @@ class CostumeAssets:
 	# description 
 	I_PET_HOUSE_9 = RuleImage(roi_front=(888,327,40,35), roi_back=(730,316,440,57), threshold=0.8, method="Template matching", file="./tasks/Component/Costume/main9/main9_pet_house_9.png")
 
+
diff --git a/tasks/Component/Costume/config.py b/tasks/Component/Costume/config.py
index 5ddb25b21..bc0f277a6 100644
--- a/tasks/Component/Costume/config.py
+++ b/tasks/Component/Costume/config.py
@@ -23,6 +23,7 @@ class MainType(str, Enum):
     COSTUME_MAIN_13 = 'costume_main_13'  # 云景阆苑
     COSTUME_MAIN_14 = 'costume_main_14'  # 雪月华庭
     COSTUME_MAIN_15 = 'costume_main_15'  # 茨心乐园
+    COSTUME_MAIN_16 = 'costume_main_16'  # 琦梦茨庭
 
 # 结界皮肤
 class RealmType(str, Enum):
diff --git a/tasks/Component/Costume/costume_base.py b/tasks/Component/Costume/costume_base.py
index d3a51ee23..7e511d7ec 100644
--- a/tasks/Component/Costume/costume_base.py
+++ b/tasks/Component/Costume/costume_base.py
@@ -25,7 +25,7 @@
         'I_HARVEST_MAIL': f'I_HARVEST_MAIL_{i}',
         'I_HARVEST_SOUL': f'I_HARVEST_SOUL_{i}',
         'I_HARVEST_GUILD_REWARD': f'I_HARVEST_GUILD_REWARD_{i}'
-    } for i in range(1, 16)
+    } for i in range(1, 17)
 }
 
 # 战斗主题（使用循环处理常规情况 + 特例处理）
diff --git a/tasks/Component/Costume/costume_test.py b/tasks/Component/Costume/costume_test.py
index 22fff51b6..6d5e30726 100644
--- a/tasks/Component/Costume/costume_test.py
+++ b/tasks/Component/Costume/costume_test.py
@@ -26,7 +26,6 @@ def run(self):
         self.ui_click(self.I_UI_BACK_YELLOW, self.I_CHECK_MAIN)
         # 宠物屋测试
         self.ui_click(self.I_PET_HOUSE, self.I_PET_CLAW)
-        self.ui_click(self.I_PET_EXIT, self.I_CHECK_MAIN)
         logger.info('Test Success')
 
 
diff --git a/tasks/Component/Costume/main16/image.json b/tasks/Component/Costume/main16/image.json
new file mode 100644
index 000000000..b98b1020e
--- /dev/null
+++ b/tasks/Component/Costume/main16/image.json
@@ -0,0 +1,47 @@
+[
+  {
+    "itemName": "check_main_16",
+    "imageName": "main16_check_main_16.png",
+    "roiFront": "1017,365,71,48",
+    "roiBack": "900,270,300,220",
+    "method": "Template matching",
+    "threshold": 0.75,
+    "description": "庭院"
+  },
+  {
+    "itemName": "main_goto_exploration_16",
+    "imageName": "main16_main_goto_exploration_16.png",
+    "roiFront": "591,183,28,46",
+    "roiBack": "520,130,190,180",
+    "method": "Template matching",
+    "threshold": 0.55,
+    "description": "探索"
+  },
+  {
+    "itemName": "main_goto_summon_16",
+    "imageName": "main16_main_goto_summon_16.png",
+    "roiFront": "966,199,23,40",
+    "roiBack": "890,130,200,180",
+    "method": 